Frogies-Project/Assets/Scripts/Core/GlobalSceneManager.cs

```diff
@@ -36,7 +36,7 @@ public class GlobalSceneManager : MonoBehaviour
         [SerializeField] private Inventory inventory;
         [SerializeField] private WaveStorage _waveStorage;
 
-        [SerializeField] private PlayerData playerData;
+        [SerializeField] public PlayerData playerData;
```

Comment:
Why do we need it public? (Haven't seen where it used)
Can you somehow get it from player instead?